ansible/inventory.yml

```diff
@@ -239,9 +239,13 @@ hosts:
             user: admin
             remote_env:
                 PATH: /opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin
-        intel-ubuntu1804-x64-1: {ip: 92.51.196.114, build_test_v8: yes, is_benchmark: yes}
-        intel-ubuntu1604-x64-2: {ip: 92.51.196.115}
-
+        intel-ubuntu1804-x64-1:
+            ip: 92.51.196.114
+            build_test_v8: yes
+            is_benchmark: yes
+        intel-ubuntu1804-x64-2:
+            ip: 92.51.196.115
+            is_benchmark: yes
```

Comment:
```suggestion
```

Reply:
I think I just understood what `is_benchmark` means (it was for the machine used to run daily benchmarks for https://benchmarking.nodejs.org/)

Reply:
Do we not need things like R, wrk, h2load, ggplot2, plyr etc that are installed by the [benchmarking role](https://github.com/nodejs/build/blob/main/ansible/roles/benchmarking/tasks/main.yml) for the [benchmark-node-micro-benchmarks](https://ci.nodejs.org/view/Node.js%20benchmark/job/benchmark-node-micro-benchmarks/) job?